### ISSUE TITLE
GAZ should not import "environmental system process" or "environmental material" and subclasses.

### ISSUE THREAD

**Issue by ddooley** [EnvironmentOntology/gaz#10]
GAZ is importing lots of ENVO, which is importing lots of FOODON?
See http://www.ontobee.org/ontology/GAZ?iri=http://purl.obolibrary.org/obo/FOODON_00002451

**Comment by pbuttigieg** [comment]
@ddooley thanks for the input, but could you make your issue title more succinct? Also, what is the actual suggestion?

**Comment by pbuttigieg** [comment]
The import chain is very strange - I'm not sure what GAZ needs with the environmental process branch at all. 

@lschriml we should branch this into another issue dedicated to cleaning up GAZ's import chain.

**Comment by ddooley** [comment]
Yes, I'm not sure how extensive the problem is, but for a start I presume "environmental system process" should not be imported into GAZ.

**Comment by ddooley** [comment]
This is still a problem, stemming from "environmental material" in OntoBee Gaz: http://www.ontobee.org/ontology/GAZ?iri=http://purl.obolibrary.org/obo/ENVO_00010483

**Comment by lschriml** [comment]
Hello Damion,
Is this causing an issue ?


Regarding GAZ, I working to revise GAZ, it has to be reduced in size by ~20% I order for it to be editable and saveable in GitHub. I’ve got the file to where I can edit it. Still working on removing duplications. 
Lots of duplicate classes and instances. 
Also looking at removing imports, splitting GAZ into two files ((1- countries, 2-oceans,/bodies of water) just to get size to be managable.

Cheers,
Lynn

**Comment by lschriml** [comment]
Agreed

**Comment by ddooley** [comment]
It isn't causing an issue at my end.  Just looking to have GAZ repaired/improved.
